app/views/admin/physical_ticket/index.html.haml

```diff
@@ -0,0 +1,41 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h2
+        Tickets Sold
+      .text-muted
+        Tickets sold for the conference
+  .row
+    .col-md-4
+      = render partial: 'admin/conferences/doughnut_chart', locals: { title: 'Tickets sold', data: @tickets_sold_distribution,}
+    .col-md-4
+      = render partial: 'admin/conferences/doughnut_chart', locals: { title: 'Tickets turnover', data: @tickets_turnover_distribution }
+  %br
+  - if @physical_tickets.any?
+    .row
+      .col-md-12
+        %table.table.table-hover.datatable#tickets
+          %thead
+            %th ID
+            %th Type
+            %th User
+            %th Actions
+          %tbody
+            - @physical_tickets.each do |physical_ticket|
+              %tr
+                %td= physical_ticket.id
+                %td= physical_ticket.ticket.title
+                %td= physical_ticket.user.name
```

Comment:
I wonder what would be more useful, I would say the email would be better. :thinking: